### PR TITLE
COMMERCE-5201 commerce-account-service fixed query

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-5201

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Review by riccardo-alberti — APPROVED** [review]

File: modules/apps/commerce/commerce-account-test/src/testIntegration/java/com/liferay/commerce/account/service/test/CommerceAccountLocalServiceTest.java [hunk 1]
```diff
@@ -213,9 +232,20 @@ public void testBusinessOrganizationCommerceAccountsVisibility()
 				CommerceAccountConstants.SITE_TYPE_B2B, StringPool.BLANK, null,
 				QueryUtil.ALL_POS, QueryUtil.ALL_POS);
 
+		int organizationUser1CommerceAccountsCount =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-account-test/src/testIntegration/java/com/liferay/commerce/account/service/test/CommerceAccountLocalServiceTest.java:235]: Lets keep the pattern the same as you had below

```suggestion
		int organizationUserCommerceAccountsCount1 =
```

File: modules/apps/commerce/commerce-account-test/src/testIntegration/java/com/liferay/commerce/account/service/test/CommerceAccountLocalServiceTest.java [hunk 2]
```diff
@@ -231,6 +261,21 @@ public void testBusinessOrganizationCommerceAccountsVisibility()
 				CommerceAccountConstants.SITE_TYPE_B2B, StringPool.BLANK, null,
 				QueryUtil.ALL_POS, QueryUtil.ALL_POS);
 
+		int organizationUser2CommerceAccountsCount =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-account-test/src/testIntegration/java/com/liferay/commerce/account/service/test/CommerceAccountLocalServiceTest.java:264]:
```suggestion
		int organizationUserCommerceAccountsCount2 =
```

File: modules/apps/commerce/commerce-account-test/src/testIntegration/java/com/liferay/commerce/account/service/test/CommerceAccountLocalServiceTest.java [hunk 3]
```diff
@@ -284,9 +329,18 @@ public void testBusinessUserCommerceAccountsVisibility() throws Exception {
 				CommerceAccountConstants.SITE_TYPE_B2B, StringPool.BLANK, null,
 				QueryUtil.ALL_POS, QueryUtil.ALL_POS);
 
+		int businessUser1CommerceAccountsCount =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-account-test/src/testIntegration/java/com/liferay/commerce/account/service/test/CommerceAccountLocalServiceTest.java:332]:
```suggestion
		int businessUserCommerceAccountsCount1 =
```

File: modules/apps/commerce/commerce-account-test/src/testIntegration/java/com/liferay/commerce/account/service/test/CommerceAccountLocalServiceTest.java [hunk 4]
```diff
@@ -301,9 +355,18 @@ public void testBusinessUserCommerceAccountsVisibility() throws Exception {
 				CommerceAccountConstants.SITE_TYPE_B2B, StringPool.BLANK, null,
 				QueryUtil.ALL_POS, QueryUtil.ALL_POS);
 
+		int businessUser2CommerceAccountsCount =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-account-test/src/testIntegration/java/com/liferay/commerce/account/service/test/CommerceAccountLocalServiceTest.java:358]:
```suggestion
		int businessUserCommerceAccountsCount2 =
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**